pkg/export/export.go

```diff
@@ -384,7 +403,7 @@ const (
 )
 
 // Run sends exported samples to Google Cloud Monitoring.
-func (e *Exporter) Run(ctx context.Context) error {
+func (e *Exporter) Run(parentCtx context.Context) error {
```

Comment:
nit: I'm fine with this reverting back to `ctx` as we're renaming the contexts used here to distinguish.